review/views.py

```diff
@@ -1,3 +1,151 @@
+import json
+
+from django.http      import JsonResponse
 from django.shortcuts import render
+from django.views     import View
+
+from review.models  import Review, ReviewLike
+from user.models    import User
+from user.utils     import id_auth
+from archive.models import Rating, Archive
+from content.models import Content
+
+
+class ReviewView(View):
+    @id_auth
+    def post(self, request, content_pk):
+        try:
+            data    = json.loads(request.body)
+            user    = request.user
+            content = Content.objects.get(id = content_pk)
+            body    = data['review']
+            
+            review, created = Review.objects.get_or_create(user    = user, 
+                                                           content = content,
+                                                           body    = body)
+
+            if not created: 
+                return JsonResponse({"message": "ALREADY_EXIST"}, status = 400)
```

Comment:
한 사용자가 하나의 리뷰만 남길 수 있는게 맞나요?
그렇다면 고유한 user_id와 content_id만 조회하는게 좋을 것 같습니다. body 내용이 조금만 다르더라도 생성될 것 같습니다.
```suggestion
            review, created = Review.objects.get_or_create(user = user, content = content)
            if created:
                review.body = data['body']
                review.save()
            else:
                return JsonResponse({"message": "ALREADY_EXIST"}, status = 400)
```

Reply:
네 말씀하신 방법으로 수정했습니다.